README.md

```diff
@@ -2,7 +2,7 @@
      title="Size Limit logo" src="docs/source/logo.png">
 # H3-Python
 
-[![Build Status](https://travis-ci.com/uber/h3-py.svg?token=8RZFXqNcjKDdLZH3myRq&branch=master)](https://travis-ci.com/uber/h3-py)
```

Comment:
This token isn't supposed to be secret, right?

Reply:
No, it's not